tasks/configure.yml

```diff
@@ -1,13 +1,23 @@
 ---
 
-- name: Set timezone
+- name: Set Timezone (trusty)
   become: yes
-  template:
-    src: timezone.j2
+  copy:
+    content: "{{ sansible_ntp_timezone }}\n"
     dest: /etc/timezone
-    owner: root
-    group: root
-    mode: 0644
+  when: ansible_distribution_release == 'trusty'
+  notify:
+    - update timezone
+
+- name: Set Timezone (xenial / bionic)
```

Comment:
Can we use the linking method on Trusty as well? Looks like we can use the same method everywhere.

Reply:
Doesn't work, I'm afraid, but I included a version check for wily and below and above.